redis/src/test/scala/zio/redis/SetsSpec.scala

```diff
@@ -1,11 +1,30 @@
 package zio.redis
 
+import scala.util.matching.Regex
+
 import zio.Chunk
+import zio.NonEmptyChunk
+import zio.ZIO
 import zio.redis.RedisError.WrongType
+import zio.stream.ZStream
 import zio.test.Assertion._
 import zio.test._
 
 trait SetsSpec extends BaseSpec {
+  private def scanAll(
+    key: String,
+    regex: Option[Regex] = None,
+    count: Option[Count] = None
+  ): ZIO[RedisExecutor, RedisError, Chunk[String]] =
+    ZStream
+      .paginateChunkM(0L) { cursor =>
+        sScan(key, cursor, regex, count).map {
+          case (nc, nm) if nc == 0 => (nm, None)
+          case (nc, nm)            => (nm, Some(nc))
+        }
+      }
+      .runCollect
```

Comment:
IMO this should be placed at the end of a trait in order for tests to be seen first while scrolling down, as they are more important than this method.